openai/openai_test.go

```diff
@@ -27,7 +29,7 @@ func TestMiddleware(t *testing.T) {
 
 	testCases := []testCase{
 		{
-			Name:     "chat handler",
+			Name:     "chat handler (default content)",
```

Comment:
```suggestion
			Name:     "chat handler",
```

can keep the name as the base case here